plugins/VersionUpgrade/VersionUpgrade42to43/VersionUpgrade42to43.py

```diff
@@ -62,10 +66,19 @@ def upgradeInstanceContainer(self, serialized: str, filename: str) -> Tuple[List
         parser["metadata"]["setting_version"] = "9"
 
         if "values" in parser:
+            for old_name, new_name in _renamed_settings.items():
+                if old_name in parser["values"]:
+                    parser["values"][new_name] = parser["values"][old_name]
+                    del parser["values"][old_name]
             for key in _removed_settings:
                 if key in parser["values"]:
                     del parser["values"][key]
 
+        parser["values"]["support_infill_angles"]["type"] = "[int]"
```

Comment:
This part is not entirely correct, parser["values"]["some_setting"] only stores the value of a setting. It doesn't store anything such as `type`, `default_value`, etc. So, here, the value needs to be updated to a list of integers.

Reply:
So instead should I write just:
parser["values"]["support_infill_angles"] = "[ ]"
?

Reply:
You can change this part to:
```
if "values" in parser:
    # Change the value from "float" to "[int]"
    if "support_infill_angles" in parser["values"]:
        old_value = float(parser["values"]["support_infill_angles"])
        new_value = int(round(old_value))
        parser["values"]["support_infill_angles"] = new_value
```

Reply:
@vgribinchuk Don't worry. I'll fix this later.

Reply:
Fixed in https://github.com/Ultimaker/Cura/pull/4598/commits/3a10345c0169a1424f02fbaa6d458f6d7a1415da

Reply:
OK, thank you!